docs/changelog.rst

```diff
@@ -2,8 +2,9 @@
 Changelog
 =========
 
-* :release:`0.4.2 <2018-08-02>`
+* :feature:`1998` Add a strategy to make sure that the account Raiden runs on always has enough balance to settle all channels. No new channels can be openend when no sufficient balance for the whole channel lifecycle is available.
 
+* :release:`0.4.2 <2018-08-02>`
```

Comment:
Are you rebased on latest master? That's already there: https://github.com/raiden-network/raiden/blob/master/docs/changelog.rst#changelog

Reply:
Yes, it just the display of the diff I think.